userpanel/modules/finances/style/bclean/templates/userbalancebox.html

```diff
@@ -47,7 +58,7 @@
 					{section name=balancelist loop=$balancelist.list step=-1}
 						{$item=$balancelist.list[balancelist]}
 						{if $item.time > $ldate}
-						<tr {if $item.value > 0} class="table-success"{else} class="table-danger"{/if}>
+						<tr class="table-click" {if $item.doctype == $smarty.const.DOC_INVOICE || $item.doctype == $smarty.const.DOC_CNOTE}data-href="?m=finances&f=invoice&id={$item.docid}"{/if} class="{if $item.value > 0}table-success{else}table-danger{/if}">
```

Comment:
2x atrybut class? To nie przejdzie - będą z tego problemy - jeśli już musisz w jednym atrybucie class złożyć wszystkie warunkowe klasy.

Reply:
ok zaraz to poprawię inaczej

Reply:
poprawione

Reply:
Ja też poprawiłem parę rzeczy w międzyczasie:
1. Nie używamy już atrybutu `type` dla elementu `script`. W html 5 nie jest potrzebny.
2. Gdy chcemy się dobrać do atrybutu `data-href` to robimy to przez `...attr("data-href")`. Dla porządku no i również dlatego, że czasami `data` może oznaczać co innego niż atrybut.
3. `onReadyState` używamy w skróconej formie - dużo wygodniej i czytelniej ;-)